core/tests/protractor/learnerFlow.js

```diff
@@ -246,29 +246,28 @@ describe('Learner dashboard functionality', function() {
     learnerDashboardPage.get();
     learnerDashboardPage.checkCompleteExplorationSection('Dummy Exploration');
     learnerDashboardPage.checkCompleteExplorationSection('Test Exploration');
-    users.logout();
+    await users.logout();
 
     // For desktop, go to the exploration editor page and
     // delete 'Dummy Exploration'.
     if (!browser.isMobile) {
       // Login as Admin and delete exploration 'Dummy Exploration'.
-      users.createAndLoginAdminUser('inspector@learnerDashboard.com',
+      await users.createAndLoginAdminUser('inspector@learnerDashboard.com',
         'inspector');
       libraryPage.get();
       libraryPage.findExploration('Dummy Exploration');
       libraryPage.playExploration('Dummy Exploration');
       // Wait for player page to completely load
       waitFor.pageToFullyLoad();
-      general.getExplorationIdFromPlayer().then(function(explorationId) {
-        general.openEditor(explorationId);
-      });
+      var explorationId = await general.getExplorationIdFromPlayer();
+      general.openEditor(explorationId);
```

Comment:
It seems to me like there are a bunch of "awaits" missing here? libraryPage.findExploration is async, general.openEditor is async, etc....

Reply:
Hi @seanlip, the learnerFlow file has not been migrated yet. @U8NWXD is working on it (based on the checklist status).

Reply:
I'm resolving this now that learnerFlow has been migrated